apps/web/src/pages/overview/components/Staking.vue

```diff
@@ -14,21 +14,28 @@ const { getEthersBalance } = useEthers()
 const { user, updateUserAgreement } = useUsers()
 const { getCurrentPrice } = usePrice()
 
-const selectedProvider = ref<ProviderString>('')
-const selectedWallet = ref(null as null | string)
-const formattedAmountToStake = ref<string>('')
+// Staking Component Refs
 const address_balance = ref(null as null | string)
 const currentEthPrice = ref<number>(0)
 const estimatedFees = ref<number|string>('-')
+const formattedAmountToStake = ref<string>('')
```

Comment:
can we try to keep a consistent way of setting the type to variables (I like using the `as type` format) if you want I can try to convert to the way you have been doing it as well. Let's talk about it at some point.

Reply:
@DemogorGod what's been pointed out as inconsistent does not come down to matter of style. Depending on the variable we will need to use different TS features for typing. The `as` keyword is used for type assertions, which provides a specific feature to assert some new type on a variable with a prior type. For example:
    `const resp = await fetch('https://...) as CustomResponseType` (if you need to override the inferred `Response` type)

Going through this list:
    18. `const addressBalance = ref<string | null>(null)` - Provide TS `string | null` in type and initialize with null
    19. `const currentEthPrice = ref(0)` - TS infers type `number` from the initial value
    20. `const estimatedFees = ref<number | string>('-')` - Good
    21. `const formattedAmountToStake = ref('')` - TS infers type `string` from the initial value

@ccali11 how come address_balance uses snake-case?

Reply:
That's how I found it. Made the changes. 👍